src/components/BarChart/BarChart-container.tsx

```diff
@@ -119,8 +119,7 @@ const BarChartContainer = (props: BarChartProps) => {
   }, [walletHash]);
 
   useEffect((): void => {
-    actions.fetchEthereumWallets();
-    actions.fetchTezosWallets();
+    actions.fetchWalletsByDataSource(dataSource);
```

Comment:
fetchWalletsByDataSource -> fetchWalletsByBlockchain
is more descriptive

Reply:
fixed